src/codeu/chat/common/User.java

```diff
@@ -104,4 +111,44 @@ private static int hash(User user) {
     hash+=user.creation.hashCode();
     return hash;
   }
+
+  public void addUserInterest(Uuid userId)
+  {
+    // This function is used to allow a User to add
+    // another user as an interest, and saves this
```

Comment:
nit: I'd avoid saying "this" unless you're using to the `this` keyword that references the current `User` instance. In other words, you're saving the parameterized `userId`, which is different from this `User` id, right?